docs/README.md

```diff
@@ -0,0 +1,23 @@
+## Generating documentation with Sphinx
+
+The best tutorial for generating documentation can be found [here](http://gisellezeno.com/tutorials/sphinx-for-python-documentation.html)
+
+Files in the source directory are generally static and should not need to be updated.  If another python module is created in `../rosette`, then the source may need to be regenerated using
+
+`sphinx-apidoc -f -o source/ ../rosette/`
```

Comment:
My preference would be to generate everything during the publish phase, but the conf.py has some default settings that need to be changed and I'd rather not sed them.  This is a compromise that seems to work.